vendor/VisualAcceptance.js

```diff
@@ -249,12 +276,12 @@ function captureHtml2Canvas (imageName, width, height, misMatchPercentageMargin,
  * @param {object} [assert=undefined] - Use only if using qunit
  * @param {string} image - Base64 image produced from html2canvas or PhantomJS
  * @param {object} browserDirectory - visual acceptance image path based off window.ui (holds browser info) and size of ember-testing-container
- * @param {object} resolve - resolve from Promise
- * @param {object} reject - reject from Promise
+ * @param {object} parentResolve - resolve from parent Promise
```

Comment:
I think we can drop `parentResolve` and `parentReject` params because now we return a real `httpGet` Promise from this method. It could simplify this method a little bit.